packages/mui-utils/package.json

```diff
@@ -26,34 +26,45 @@
     "url": "https://opencollective.com/mui"
   },
   "scripts": {
-    "build": "yarn build:legacy && yarn build:modern && yarn build:node && yarn build:stable && yarn build:types && yarn build:copy-files",
+    "build": "pnpm build:legacy && pnpm build:modern && pnpm build:node && pnpm build:stable && pnpm build:types && pnpm build:copy-files",
     "build:legacy": "node ../../scripts/build.mjs legacy",
     "build:modern": "node ../../scripts/build.mjs modern",
     "build:node": "node ../../scripts/build.mjs node",
     "build:stable": "node ../../scripts/build.mjs stable",
     "build:copy-files": "node ../../scripts/copyFiles.mjs",
     "build:types": "node ../../scripts/buildTypes.mjs",
     "prebuild": "rimraf build tsconfig.build.tsbuildinfo",
-    "release": "yarn build && npm publish build",
+    "release": "pnpm build && npm publish build",
     "test": "cd ../../ && cross-env NODE_ENV=test mocha 'packages/mui-utils/**/*.test.{js,ts,tsx}'",
     "typescript": "tslint -p tsconfig.json \"{src,test}/**/*.{spec,d}.{ts,tsx}\" && tsc -p tsconfig.json"
   },
-  "peerDependencies": {
-    "react": "^17.0.0 || ^18.0.0"
-  },
   "dependencies": {
     "@babel/runtime": "^7.22.6",
-    "@types/prop-types": "^15.7.5",
-    "@types/react-is": "^18.2.1",
     "prop-types": "^15.8.1",
     "react-is": "^18.2.0"
   },
   "devDependencies": {
     "@babel/helper-module-imports": "^7.22.5",
+    "@mui/utils": "workspace:*",
```

Comment:
I have no idea what's going on here, but it's the only way I could make it work. Without this, it was not possible to import from '@mui/utils'. I haven't found any substantial differences between this package and all others (in which self-referencing work fine).